### DISCUSSION TITLE
Noob Needs Help

### DISCUSSION THREAD

**Discussion by shaban12** [hyunbinseo/zoom-rec-dl#9] [Q&A]
There have been some pricing issues with zoom and they will terminate my recordings on 24th Feb, There are over 250 recordings in my account. I don't know how to get all of them backed up and I am not aware of the code that much either.
I can get around the code but I am stuck on how would I be able to get the password-embedded links for all of the recordings.
Please ignore my stupidity.

**Comment by hyunbinseo** [comment]
I receive pre-generated Zoom cloud sharing links from another source.

Therefore, I do not know much about the URL generation process itself.

Are you trying to generate the links in bulk?

---

> 1. [Access your cloud recordings](https://support.zoom.us/hc/en-us/articles/205347605#h_9898497b-e736-4980-a749-d55608f10773).
> 2. Find the cloud recording you want to share, download or delete.
>     1. On the recording's page, click Share.
>     2. Adjust your [sharing settings](https://support.zoom.us/hc/en-us/articles/205347605#h_750e7e94-45e4-415b-a67a-ac8b49e74fa2) as desired.
>     3. Click Copy Sharing Information, then navigate to your email or where you need to share the cloud recording, to paste the sharing information.

— [Managing and sharing cloud recordings](https://support.zoom.us/hc/en-us/articles/205347605)

**Reply by shaban12** [reply]
> I receive pre-generated Zoom cloud sharing links from another source.
> 
> Therefore, I do not know much about the URL generation process itself.
> 
> Are you trying to generate the links in bulk?
> 
> > 1. [Access your cloud recordings](https://support.zoom.us/hc/en-us/articles/205347605#h_9898497b-e736-4980-a749-d55608f10773).
> > 2. Find the cloud recording you want to share, download or delete.
> >    
> >    1. On the recording's page, click Share.
> >    2. Adjust your [sharing settings](https://support.zoom.us/hc/en-us/articles/205347605#h_750e7e94-45e4-415b-a67a-ac8b49e74fa2) as desired.
> >    3. Click Copy Sharing Information, then navigate to your email or where you need to share the cloud recording, to paste the sharing information.
> 
> — [Managing and sharing cloud recordings](https://support.zoom.us/hc/en-us/articles/205347605)

Yes I am trying to generate in bulk, Because I need to back up those recordings just in case Zoom decides to permanently delete those from their server.

**Comment by hyunbinseo** [comment]
I believe you should use an utility that utilizes the Zoom API.

> **Warning**
> I am not the developer of the following tool, and I do not take any responsibility regarding it.

An example would be [zoom-recording-downloader](https://github.com/ricardorodrigues-ca/zoom-recording-downloader). Reference the related [Reddit post](https://www.reddit.com/r/Zoom/comments/109izns/how_to_bulk_batch_export_download_all_recordings/?utm_source=share&utm_medium=web2x&context=3).

**Reply by shaban12** [reply]
> I believe you should use an utility that utilizes the Zoom API.
> 
> > Warning
> > I am not the developer of the following tool, and I do not take any responsibility regarding it.
> 
> An example would be [zoom-recording-downloader](https://github.com/ricardorodrigues-ca/zoom-recording-downloader). Reference the related [Reddit post](https://www.reddit.com/r/Zoom/comments/109izns/how_to_bulk_batch_export_download_all_recordings/?utm_source=share&utm_medium=web2x&context=3).

Hey thank you for that, This is working but is working to download just Audio. I need the Video file. Can you guide me there?
I have started download the download and saw that it is downloading audio only. Your help is greatly appreaciated. 
![image](https://user-images.githubusercontent.com/37975354/219854520-49183adb-4bd0-45f7-9315-15d9b3d977ce.png)

**Reply by hyunbinseo** [reply]
The script does not support audio-only download feature. https://github.com/ricardorodrigues-ca/zoom-recording-downloader/issues/13

Have you tried to download each recording manually from the Zoom portal? or via my script?

**Reply by shaban12** [reply]
The script you shared earlier zoom-recording-downloader, I ran that and it is downloading audio only.
Or maybe this whole 
![image](https://user-images.githubusercontent.com/37975354/219855022-9b806ee8-374f-4ac9-b11d-8314c1470d93.png)
Process ends and then the video downloading starts?
I need videos.

**Reply by hyunbinseo** [reply]
I cannot support you with another script.

I have not used [zoom-recording-downloader](https://github.com/ricardorodrigues-ca/zoom-recording-downloader).

**Reply by shaban12** [reply]
The one you provided, Zoom Recording downloader #13. Will this download Videos Only?

**Reply by hyunbinseo** [reply]
It seems to download all formats. https://github.com/ricardorodrigues-ca/zoom-recording-downloader/issues/29

```


                               ,*****************.
                            *************************
                          *****************************
                        *********************************
                       ******               ******* ******
                      *******                .**    ******
                      *******                       ******/
                      *******                       /******
                      ///////                 //    //////
                       ///////*              ./////.//////
                        ////////////////////////////////*
                          /////////////////////////////
                            /////////////////////////
                               ,/////////////////

                           Zoom Recording Downloader

                                  Version 2.1

Log file not found. Creating new log file

Getting user accounts...

Getting recording list for
==> Found 277 recordings==> Downloading (1 of 277) as audio_only
==> Downloading (1 of 277) as shared_screen_with_speaker_view
==> Downloading (2 of 277) as shared_screen_with_gallery_view
==> Downloading (2 of 277) as shared_screen_with_speaker_view
==> Downloading (2 of 277) as chat_file
==> Downloading (2 of 277) as audio_only
==> Downloading (3 of 277) as audio_only
==> Downloading (3 of 277) as shared_screen_with_speaker_view
==> Downloading (4 of 277) as audio_only
==> Downloading (4 of 277) as shared_screen_with_speaker_view
==> Downloading (5 of 277) as shared_screen_with_speaker_view
==> Downloading (5 of 277) as audio_only
==> Downloading (6 of 277) as audio_only
==> Downloading (6 of 277) as shared_screen_with_speaker_view
==> Downloading (7 of 277) as shared_screen_with_speaker_view
```
